services/ui-src/src/components/app/AppRoutes.tsx

```diff
@@ -11,7 +11,7 @@ export const AppRoutes = () => {
         <Route path="/" element={<HomePage />} />
         <Route path="/profile" element={<ProfilePage />} />
         <Route path="/help" element={<HelpPage />} />
-        <Route path="/dashboard" element={<DashboardPage />} />
```

Comment:
i think something is missing, as if you do this and log in as a stateuser, we lost the dashboard page for the state.user

https://github.com/user-attachments/assets/a7cdac03-c5c8-43c8-a87a-29f343204de3

Reply:
actually, this might be solved with Ben's pr changes. see if pulling the latest fixes this?

Reply:
Yes, just pulled his changes into the branch and it works now 😄